File: app/src/main/java/pro/eugw/lessoncountdown/MHolder.java
package pro.eugw.lessoncountdown;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

class MHolder extends RecyclerView.ViewHolder {

    TextView name;
    TextView time;

    MHolder(View itemView) {
        super(itemView);
        name = itemView.findViewById(R.id.textName);
        time = itemView.findViewById(R.id.textTime);
    }

}

File: app/src/main/java/pro/eugw/lessoncountdown/list/schedule/MHolder.kt
package pro.eugw.lessoncountdown.list.schedule

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import pro.eugw.lessoncountdown.R

internal class MHolder(itemView: View) : androidx.recyclerview.widget.RecyclerView.ViewHolder(itemView) {

    val lesson: TextView = itemView.findViewById(R.id.textLesson)
    val time: TextView = itemView.findViewById(R.id.textTime)
    val imageHomework: ImageView = itemView.findViewById(R.id.imageHomework)
    val layout: View = itemView.findViewById(R.id.layoutAll)

}
